src/bin/casr-python.rs

```diff
@@ -0,0 +1,310 @@
+                let report_end = report_end + 1;
+                if report_end > python_stdout_list.len() {
+                    bail!("Corrupted output");
+                }
+                report.python_report = Vec::from(&python_stdout_list[report_start..report_end]);
+
+                // Get exception from python report.
+                if let Ok(exception) = python_exception(&report.python_report[1]) {
+                    report.execution_class = exception;
+                }
+
+                // Get stack trace from python report.
+                let first = report
+                    .python_report
+                    .iter()
+                    .position(|line| line.starts_with("Traceback "));
+                if first.is_none() {
+                    bail!("Couldn't find traceback in python report");
+                }
+
+                // Stack trace is splitted by empty line.
+                let first = first.unwrap();
+                let last = report
+                    .python_report
+                    .iter()
+                    .skip(first)
+                    .rposition(|s| !s.is_empty());
+                if last.is_none() {
+                    bail!("Couldn't find traceback end in python report");
+                }
+                let last = last.unwrap();
+
+                let re = Regex::new(
+                    r#"(File ".+", line [\d]+, in .+|\[Previous line repeated (\d+) more times\])"#,
+                )
+                .unwrap();
+                report.stacktrace = report.python_report[first..first + last]
+                    .iter()
+                    .rev()
+                    .map(|s| s.trim().to_string())
+                    .filter(|s| re.is_match(s))
+                    .collect::<Vec<String>>();
+            } else {
+                bail!("Corrupted output");
+            }
+        } else {
+            bail!("Need to use casr-san with LD_PRELOAD")
+        }
+    } else if let Some(report_start) = python_stderr_list
+        .iter()
+        .position(|line| line.contains("Traceback "))
+    {
+        // Set python report in casr report.
+        if let Some(report_end) = python_stderr_list.iter().rposition(|s| !s.is_empty()) {
+            let report_end = report_end + 1;
+            if report_end > python_stderr_list.len() {
+                bail!("Corrupted output");
+            }
+            report.python_report = Vec::from(&python_stderr_list[report_start..report_end]);
+
+            let last = report
+                .python_report
+                .iter()
+                .skip(report_start)
+                .rposition(|s| !s.is_empty());
+            if last.is_none() {
+                bail!("Couldn't find traceback end in python report");
+            }
+            let last = last.unwrap();
+
+            let re =
+                Regex::new(r#"(File "[\w\-<>~./]+", line [\d]+, in [\w<>]+|\[Previous line repeated (\d+) more times\])"#).unwrap();
```

Comment:
Maybe use similar regex as in python.rs?

Reply:
No, they're different

Reply:
I mean simplifying `[\w\-<>~./]+` to `.+`

Reply:
Done